project: listeners management documentation page (#550)
mia-platform/documentation: docs/development_suite/api-console/api-design/endpoints.md

> 

Reviewer: With the introduction of listeners management, in this documentation page, we have also to change the image and the information about endpoint creation. See the following paragraph https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/endpoints#details

Moreover, even info about required fields in endpoint creation has to be updated by adding listeners field info

Reviewer 2: updated screenshots

## Endpoint settings

From this section it is possible to configure: _Listeners_, _Security_, _Documentation_, _Microservice Gateway_ and _Advanced_ settings for the Endpoint.

### Listeners

![Endpoint settings card - Listeners](img/listeners/endpoint-configure-listeners.png)

This section lets you define from which Listener your Endpoint should be exposed.  

> Keep in mind that you must expose and Endpoint with _at least_ one Listener.

Reviewer 3: ```suggestion
Keep in mind that you must expose an Endpoint with _at least_ one Listener.
```

> Keep in mind that it is not possible to leave an Endpoint not exposed by any Listener, thus every Endpoint must be exposed by _at least one_ Listener.

Reviewer 4: ```suggestion
Keep in mind that you must expose and Endpoint with _at least_ one Listener.
```
